content/docs/node-operators/_index.md

```diff
@@ -1,27 +1,24 @@
 ---
 title: "Node Operators"
 linkTitle: "Node Operators"
-weight: 3
+weight: 40
 description: >
   How to setup and operate an Autonity Go Client node on an Autonity testnet
 ---
 
-This section outlines how to:
+All nodes on the Autonity network exchange blocks and transactions with other nodes, keep track of the current start of the blockchain.  Running a node offers a dedicated RPC endpoint for querying and for accepting transactions.  DApps running on the network may take advantage of dedicates nodes for any "backend" or off-chain operations.
```

Comment:
I think this was the intended meaning?
```suggestion
All nodes on the Autonity network exchange blocks and transactions with other nodes, keeping track of the current state of the blockchain.  Running a node offers a dedicated RPC endpoint for querying and for accepting transactions.  DApps running on the network may take advantage of dedicates nodes for any "backend" or off-chain operations.
```

Reply:
Yes, but additional edit: should be dedicated:

>advantage of dedicates nodes -> advantage of dedicated nodes

Reply:
updated